config/README.md

```diff
@@ -29,6 +29,8 @@ You can search for feature flags by partial string or regular expression with th
 
 Run `yarn feature-search [search]` from the root calypso directory to see example searches.
 
+You can also activate feature flags only in your development environment. To do this, create a `.env` file in the root folder of the cloned repository. Add the variable `ACTIVE_FEATURE_FLAGS`, and specify the feature flags you want to activate, separated by commas. For more details, see https://github.com/motdotla/dotenv.
```

Comment:
I wonder if this should link to https://github.com/mrsteele/dotenv-webpack instead. When I went to the existing link, the config didn't seem to be anything that was added in this PR.

Reply:
Changed here: https://github.com/Automattic/wp-calypso/pull/91908/commits/7699de31f48efd540042cc699117fbfa11231359

Since it was just the readme, I already merged it. But if you have any other observations, I can open a new PR. 😉